plugins/woocommerce/src/Admin/API/Reports/TimeInterval.php

```diff
@@ -79,28 +79,29 @@ public static function default_after() {
 	 *
 	 * @param string $time_interval Time interval.
 	 * @param string $table_name Name of the db table relevant for the date constraint.
+	 * @param string $date_column_name Name of the date table column.
 	 * @return mixed
 	 */
-	public static function db_datetime_format( $time_interval, $table_name ) {
+	public static function db_datetime_format( $time_interval, $table_name, $date_column_name ) {
```

Comment:
```suggestion
	public static function db_datetime_format( $time_interval, $table_name, $date_column_name = 'date_created' ) {
```

Could we add a default value for `$date_column_name` for backward compatibility?

Reply:
Great catch, @chihsuan! 🎉